### PR TITLE
Add Deprecation Notice

### PR DESCRIPTION
**Pull request by davidkhala**
Per explanation from Bret Logan

Signed-off-by: davidliu <david-khala@hotmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,15 +6,22 @@ The purpose of this baseimage is to act as a bridge between a raw ubuntu/xenial
 
 Therefore, the expensive FOSS components are built into this baseimage once and subsequently cached on the public repositories so that workflows may simply consume the objects without requiring a local build cycle.
 
-# Intended Audience
+## Deprecation Notice
+
+Baseimage is essentially retired already. There are several months left on the Fabric 1.4 LTS and then we will archive the repo. 
+
+None of those images in Baseimage are used in Fabric release-2.x anymore
```
Comment (lindluni) [inline README.md]: Thanks David for doing this, but can we change it to something like this:

```
The Fabric-Baseimage repository and the images built from this repository are deprecated. Critical bugs will continue to be patched through the Fabric 1.4.x LTS lifecycle. The images affected by this deprecation are:

- hyperledger/fabric-baseimage (with a version < 1.0.0)
- hyperledger/fabric-baseos (with a version < 1.0.0)
- hyperledger/fabric-couchdb (all versions)
- hyperledger/fabric-zookeeper (all versions)
- hyperledger/fabric-kafka (all versions)

Users should migrate to open source versions of these images.
```

Comment (lindluni) [reply README.md]: Can we also put the deprecation notice as the first item in the README

**Review by lindluni — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (davidkhala) [reply README.md]: Definitely. WIP

**Review by lindluni — APPROVED** [review]

**Comment by lindluni** [comment]
Thank you!

**Pull request merged**